src/main/java/com/github/pires/obd/commands/control/DtcNumberCommand.java

```diff
@@ -45,8 +45,7 @@ protected void performCalculations() {
      */
     public String getFormattedResult() {
         final String res = milOn ? "MIL is ON" : "MIL is OFF";
-        return new StringBuilder().append(res).append(codeCount).append(" codes")
-                .toString();
+        return res + codeCount + " codes";
```

Comment:
didnt you shoot down a pull request of mine due to string builder efficiencies or something like that?